improver_tests/nbhood/circular_kernel/test_GeneratePercentilesFromACircularNeighbourhood.py

```diff
@@ -816,20 +746,7 @@ def test_circle_bigger_than_domain(self):
         radius = 50000.0
         msg = "Distance of {}m exceeds max domain distance".format(radius)
         with self.assertRaisesRegex(ValueError, msg):
-            GeneratePercentilesFromACircularNeighbourhood().run(self.cube, radius)
-
-    def test_mask_cube(self):
```

Comment:
With the refactoring, there is no need to have the same interface between this plugin and other plugins, so I've removed passing in a mask_cube as an option. We now don't need to test what happens if you accidentally pass one in.